### PR TITLE
Change to connect to tapyrus chain.

### PR DESCRIPTION
**Pull request by rantan**
Before this PR, this implementation connects to bitcoin node. This PR changes to connect tapyrus node for to implement tapyrus specific functions.

### CODE REVIEW

**Review by azuchi — COMMENTED** [review]
`db_chain_store.rs` does not seem to be necessary.

File: src/network/utils/codec.rs [hunk 1]
```diff
@@ -43,6 +44,19 @@ impl Decoder for NetworkMessagesCodec {
...
+
+                // Skip unrecognized message.
+                src.advance(4 + 12); // magic(4bytes) + command string(12bytes)
+                let payload_size = {
+                    let mut decoder = io::Cursor::new(&src);
+                    ReadBytesExt::read_u32::<LittleEndian>(&mut decoder)? as usize
+                };
+                src.advance(4 + 4 + payload_size); // length(4bytes) + checksum(4bytes) + payload
+
+                Ok(None)
```
Comment (azuchi) [inline src/network/utils/codec.rs:60]: If this happens, the counter party may be executing a command that is not defined in the network protocol. In that case, I think that it is better not to skip, but to disconnect such a node.

Comment (rantan) [reply src/network/utils/codec.rs:60]: rust-tapyrus  still has unsupporting messages which defined in network protocol like `cmpctblock`, `feefilter`. So we can't decode these message. I made it just skip these messages so far.

**Comment by rantan** [comment]
> `db_chain_store.rs` does not seem to be necessary.

File: src/network/utils/codec.rs (hunk 1 above)
Comment (azuchi) [reply src/network/utils/codec.rs:60]: I see. please comment it.

**Comment by rantan** [comment]
@azuchi  I updated the points you mentioned.

**Pull request merged**